src/components/common/form/message-item-types/DoubleDateMoved.jsx

```diff
@@ -0,0 +1,51 @@
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import { formatDateNoYear } from '../../../../utils/time-entry-utils/timeEntryUtils';
+
+const DoubleDateMoved = ({
+  variables,
+  setSummaryMessages,
+  setIsAlertVisible,
+}) => {
+  const clearSummary = () => {
+    setSummaryMessages({});
+    setIsAlertVisible(false);
+  };
+
+  const startDate = variables[0];
+  const endDate = variables[1];
```

Comment:
Could these be in an object instead of array?

Reply:
Updated to be an object